### PR TITLE
Couple fixes

### LINKED ISSUE #178: Two context menus are shown at the same time
When you right click on mouse to delete a point from a polygon then two context menus are shown at the same time. Thus it is not possible to delete such point using context menu.

### LINKED ISSUE #179: Sometimes attributes can be truncated in attribute annotation mode
See on the screenshot below. In some cases (near image borders) attributes can be truncated and it is very difficult to annotate such objects in attribute annotation mode.

image

### PR DESCRIPTION
**Pull request by bsekachev**
Resolved #178 
Resolved #179

* Fixed: Sometimes attributes can be truncated. Dragging, resizing and drawing processes was improved 
* Fixed: both context menu are opened simultaneously 
* Fixed: bug with propagate when frames have different size
* Improvement: auto fit after AAM

### CODE REVIEW

File: cvat/apps/engine/static/engine/js/player.js [hunk 1]
```diff
@@ -123,6 +123,7 @@ class FrameProvider extends Listener {
 
 const MAX_PLAYER_SCALE = 10;
 const MIN_PLAYER_SCALE = 0.1;
+const PLAYER_FRAME_OFFSET = 1000;
```
Comment (nmanovic) [inline cvat/apps/engine/static/engine/js/player.js]: PLAYER_FRAME_OFFSET doesn't look as a good solution. It is a workaround for mention problems but may be it is a whole in GUI design. Let's find a solution without PLAYER_FRAME_OFFSET.

Comment (nmanovic) [reply cvat/apps/engine/static/engine/js/player.js]: One more possible solution is to emphasize the drawing area visually (white paper).

File: cvat/apps/engine/static/engine/js/annotationUI.js [hunk 2]
```diff
@@ -737,8 +739,8 @@ function translateSVGPos(svgCanvas, clientX, clientY) {
     pt = pt.matrixTransform(svgCanvas.getScreenCTM().inverse());
 
     let pos = {
-        x: pt.x,
-        y: pt.y
+        x: pt.x - PLAYER_FRAME_OFFSET,
```
Comment (nmanovic) [inline cvat/apps/engine/static/engine/js/annotationUI.js]: Need to have corresponding methods to calculate coordinates from/to different coordinate systems.

File: cvat/apps/engine/static/engine/js/player.js (hunk 1 above)
Comment (nmanovic) [inline cvat/apps/engine/static/engine/js/player.js]: Let's keep drawing canvas with fixed size (image_w + 2*offset, image_h + 2*offset). Player fit should focus on the image itself as previously

Comment (bsekachev) [reply cvat/apps/engine/static/engine/js/player.js]: Done

File: cvat/apps/engine/static/engine/js/annotationUI.js (hunk 2 above)
Comment (bsekachev) [reply cvat/apps/engine/static/engine/js/annotationUI.js]: Done

File: cvat/apps/engine/static/engine/js/player.js (hunk 1 above)
Comment (bsekachev) [reply cvat/apps/engine/static/engine/js/player.js]: Done

File: cvat/apps/engine/static/engine/js/shapeBuffer.js [hunk 3]
```diff
@@ -298,35 +293,37 @@ class ShapeBufferView {
...
+        let points = this._shape.position.points ? this._shape.position.points.split(' ')
+            .map((coord) => coord.split(',').map((x) => +x + PLAYER_FRAME_OFFSET)).join(' ') : null;
 
         switch (this._shape.type) {
         case 'box': {
             let width = this._shape.position.xbr - this._shape.position.xtl;
             let height = this._shape.position.ybr - this._shape.position.ytl;
             this._shapeView = this._frameContent.rect(width, height)
-                .move(this._shape.position.xtl, this._shape.position.ytl)
+                .move(this._shape.position.xtl + PLAYER_FRAME_OFFSET, this._shape.position.ytl + PLAYER_FRAME_OFFSET)
```
[the remark this answers is not in the document]
Comment (bsekachev) [reply cvat/apps/engine/static/engine/js/shapeBuffer.js]: Done

File: cvat/apps/engine/static/engine/js/shapeBuffer.js [hunk 4]
```diff
@@ -344,10 +341,13 @@ class ShapeBufferView {
 
     _moveShapeView(pos) {
         let rect = this._shapeView.node.getBBox();
-        this._shapeView.move(pos.x - rect.width / 2, pos.y - rect.height / 2);
+        let x = pos.x + PLAYER_FRAME_OFFSET;
```
[the remark this answers is not in the document]
Comment (bsekachev) [reply cvat/apps/engine/static/engine/js/shapeBuffer.js]: Done

File: cvat/apps/engine/static/engine/js/shapeBuffer.js [hunk 5]
```diff
@@ -298,35 +293,37 @@ class ShapeBufferView {
 
     _drawShapeView() {
         let scale = window.cvat.player.geometry.scale;
+        let points = this._shape.position.points ? this._shape.position.points.split(' ')
+            .map((coord) => coord.split(',').map((x) => +x + PLAYER_FRAME_OFFSET)).join(' ') : null;
```
[the remark this answers is not in the document]
Comment (bsekachev) [reply cvat/apps/engine/static/engine/js/shapeBuffer.js]: Done

File: cvat/apps/engine/static/engine/js/coordinateTranslator.js [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+                }
+
+                if ("y" in canvasBox) {
+                    canvasBox.y += this._playerOffset;
+                }
+
+                return canvasBox;
+            },
+
+            canvasToActual: function(canvasBox) {
```
[the remark this answers is not in the document]
Comment (bsekachev) [reply cvat/apps/engine/static/engine/js/coordinateTranslator.js]: Done

File: cvat/apps/engine/static/engine/js/coordinateTranslator.js [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+                        });
+                    }
+                    return canvasPoints;
+                }
+                else {
+                    throw Error('Unknown points type was found');
+                }
+            },
+
+            canvasToActual: function(canvasPoints) {
```
[the remark this answers is not in the document]
Comment (bsekachev) [reply cvat/apps/engine/static/engine/js/coordinateTranslator.js]: Done

**Pull request merged**